packages/integration-tests/Executions.js

```diff
@@ -69,6 +69,7 @@ const findExecutionArn = async (prefix, matcher, queryParameters = { }, options
     },
     {
       retries: options.timeout,
+      minTimeout: 1000,
```

Comment:
Line 25 was misleading, because that was not true. Timeout is the number of times, but this makes it true. I changed this because I didn't see a real reason we need to hit the executions more than once per second.